app/controllers/concerns/curation_concerns/collections_controller_behavior.rb

```diff
@@ -13,6 +13,9 @@ module CollectionsControllerBehavior
 
       # This is needed as of BL 3.7
       copy_blacklight_config_from(::CatalogController)
+      configure_blacklight do |config|
+        config.search_builder_class = CurationConcerns::CollectionSearchBuilder
```

Comment:
But it never references that variable, so setting it is just code that isn't needed.